tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
this would be 2 tsconfig files. Can we combine into the one we already have in the src folder as that is the root of the ts project?